lib/bootloader_setup.pm

```diff
@@ -697,7 +697,7 @@ sub specific_bootmenu_params {
         diag "Disabling installer self update";
         push @params, "self_update=0";
     }
-    elsif (my $self_update_repo = get_var("INSTALLER_SELF_UPDATE")) {
+    elsif (my $self_update_repo = get_var("INSTALLER_SELF_UPDATE", '1')) {
```

Comment:
so if the job does not set setting 'INSTALLER_SELF_UPDATE', '1' would be used as linuxrc parameter (which is not valid one, only is valid the url if activated). I think it doesn't make sense to add the default and might break something.

Reply:
+1 here, this variable expects URL and is not boolean value, so it will actually break things when variable is not set.

Reply:
Ok, I got confused here as we have 2 settings. One is `self_update` and another one is `self_update_url` https://documentation.suse.com/sles/15-SP1/html/SLES-all/appendix-linuxrc.html 
To be honest, I thought you cannot put URL in the `self_update`, but in your VR seems that it worked.

Reply:
https://documentation.suse.com/sles/15-SP2/html/SLES-all/appendix-linuxrc.html

Reply:
The self_update takes 0 or uri. 1 is the default but has no impact. it just the default behavior. We use **1** as now. the change is for *if url is not given, keep using the 1* for the self_update scenario

Reply:
then, we should not write in boot parameters an option that is wrong: `self_update=1`, you are pushing it always below.

Reply:
But this is what we already do and i kept it for not cause any issues and keep it compatible with the existing one.

Reply:
I don't get your point, doesn't add this a new linuxrc for other cases? could you try other situation different than this one or explain it better, please.
I don't see much consistency in the way is used, here there is not default: https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/10864/files#diff-4b8e4ff32cbe625eacfff10545baad63R109
I think reverting the line would be the most consistent way. wdyt?

Reply:
the `self_update=1` is the default setting in all the installation unless explicitly you say `self_update=0`. add it or not it makes any difference. is it clear?

Reply:
After a discussion with @jknphy we agreed that this is unnecessary. The test should behave the same for self_update but also for any other installation even without. I reverted this line